### ISSUE TITLE
ReBeat clock dividers unsync on change

### ISSUE THREAD

**Issue by WouterSpekkink** [AScustomWorks/Paid-Modules-Info#4]
I figured out what my issue with ReBeat was (referenced in other issue I posted): If I set the various clock dividers to different settings, then they start to get out of sync. For example, when I first set a simple 4-to-4 beat for a kick on the first row (setting it to 4), and then use the second row to trigger a snare on every second kick (setting it to 8), then the two triggers actually do not fall on the same beat pattern unless I reset the whole module. In other words, it is impossible to make changes on the fly without unsyncing the dividers. 

It would also be great to have a run input on this module.

**Comment by AScustomWorks** [comment]
Hi WouterSpekkink, I really didn't thought about on the fly changes with this module, indeed making changes on the fly sometimes can get the beat to be syncopated but not out of sync with the clock source though. (kind of an undesired but nice side effect if you ask me hehe).
I'll have to think about a way to force the clocks to stay in sync with each other, with a switch to go back to the previous behavior if wanted.

About the  run input, sure, it can be added, but only on ReBeat as there's no space available on ReBit to squeeze it ;)

**Comment by WouterSpekkink** [comment]
Right, the sometimes nice side-effect is also what I noticed. It would be great though to also have the option to make changes on the fly without having this side-effect, so adding a switch to back and forth between the two types of behaviour sounds like a good solution to me. The main reason is that I like the idea of improvisation 'on the fly', and the ReBeat would be a really great tool in that.
